### DISCUSSION TITLE
Add support for secrets/environment variables

### DISCUSSION THREAD

**Discussion by andreifloricel** [codesandbox/sandpack#403] [General]
Unless I missed something, currently it is NOT possible to pass some secrets as environment variables (similar to https://codesandbox.io/docs/secrets)

As a real use case, I would like to provide some Firebase keys without exposing them.

**Comment by danilowoz** [comment]
Sandpack doesn't support environment variables or secrets due to technical limitations. Supporting this in the future would likely require logins and a paid service tier.
